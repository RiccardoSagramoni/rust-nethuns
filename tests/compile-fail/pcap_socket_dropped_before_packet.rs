// use nethuns::sockets::Local;
// use nethuns::sockets::pcap::NethunsSocketPcap;
// use nethuns::types::NethunsSocketOptions;

// fn main() {
//     let opt = NethunsSocketOptions::default();
    
//     let socket = NethunsSocketPcap::<Local>::open(opt, "filename", false).unwrap();
    
//     let p1 = socket.read().unwrap();
//     let p2 = socket.read().unwrap();
//     println!("{:?} {:?}", p1, p2);
//     drop(socket);
//     drop(p1);
// }

// TODO
