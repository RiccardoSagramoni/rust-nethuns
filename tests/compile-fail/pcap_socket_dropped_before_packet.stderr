error[E0601]: `main` function not found in crate `$CRATE`
  --> tests/compile-fail/pcap_socket_dropped_before_packet.rs:17:9
   |
17 | // TODO
   |         ^ consider adding a `main` function to `$DIR/tests/compile-fail/pcap_socket_dropped_before_packet.rs`
